server/src/main/java/org/opensearch/autotagging/RuleValidator.java

```diff
@@ -80,6 +98,10 @@ private boolean isNullOrEmpty(String str) {
         return str == null || str.isEmpty();
     }
 
+    private boolean isEmpty(String str) {
```

Comment:
we should rename this method since it is missing the context. Maybe `isInvalidUpdatedValue` makes more sense here ?